components/Graph.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState, useEffect } from 'react';
+import { View, TouchableOpacity, Dimensions } from 'react-native';
+import { LineChart } from 'react-native-gifted-charts';
+import { FText } from '~/components/Text/FText';
+
+interface DataPoint {
+  value: number;
+  label: string;
+}
+
+const Graph = () => {
+  // Generate mock data only once
+  // const [allData] = useState<DataPoint[]>(
+  //   Array.from({ length: 365 }, (_, i) => ({
+  //     value: Math.floor(Math.random() * (2763 - 1000 + 1)) + 1000, // Random price
+  //     label: new Date(Date.now() - i * 24 * 60 * 60 * 1000).toISOString(), // ISO date
+  //   }))
+  // );
```

Comment:
No problem in keeping comments like this to test the component in the file but can you put them at the bottom of the file to keep things clean